package.json

```diff
@@ -132,7 +132,8 @@
   },
   "dependencies": {
     "bluebird": "^3.5.1",
-    "clappr": "0.2.78",
+    "clappr": "^0.2.78",
```

Comment:
Sorry, can you do `--save-exact` for clappr. This line shouldn't change at all then

Reply:
ok